### DISCUSSION TITLE
How to create admin account with disabled signup?

### DISCUSSION THREAD

**Discussion by CRASH-Tech** [ngoduykhanh/PowerDNS-Admin#1578] [Q&A]
I no need signup function at all, only one static admin. 
And I need to enable signup, creaate user and disable signup again.

**Comment by AzorianMatt** [comment]
@CRASH-Tech currently, the first admin account must be created this way, no exceptions.

**Comment by jbe-dw** [comment]
Hello,

On fresh install, the 1st user is the admin. You can create this user then disable signup. On automated install, this is done via the database.

Regards
